README.md

```diff
@@ -94,6 +94,9 @@ detecting those extension as follows, if needed:
 let g:yagpdbcc_override_ft = 1
 ```
 
+Note that this particular bit of setup must come **before** the plugin is loaded (prpbobly by your plugin manager),
```

Comment:
Unless I am misinterpreting, this feels rather redundant for the following reasons:

Globals are always read before anything else is sourced, given by the precedence that `vimrc` and related files have.
I also believe that the average (N)vim user knows to reload or restart their editor.

If I am misinterpreting this, then consider changing it to be more clear, or remove it due to its redundant nature.

Reply:
Are globals always read first? I get different behavior depending on whether my `init.vim` sets `g:yagpdbcc_override_ft` before or after plugin loading.

Reply:
Okay, so I was just misinterpreting then, no worries :+1: